Makefile

```diff
@@ -34,9 +34,8 @@ test:
 
 integration-test:
 	@echo "=== $(INTEGRATION) === [ test ]: running integration tests..."
-	@docker-compose -f tests/integration/docker-compose.yml up -d --build
-	@go test -v -tags=integration ./tests/integration/. || (ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
-	@docker-compose -f tests/integration/docker-compose.yml down
+	@go clean -testcache
+	@go test -v -timeout 600s -tags=integration ./tests/integration/. || (ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
```

Comment:
is 
```
(ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
```
a leftover?

Reply:
I was thinking that in case in the future for whatever reason the suite fails to complete (panic, manually kill the process, etc) at the end we try to do docker-compose down